### ISSUE TITLE
I have not heard any of the participants yet started

### ISSUE THREAD

**Issue by prolay7** [gtainfotech/goforit_web#2]
Let me know when the project can be started any of the participants

**Comment by gtainfotech** [comment]
Please note that frequency of positive contributions will play a role in determining your hourly rate paid to you, as your reward, once the app is running. You can ask me your hourly rate privately. Also it helps to accurately gauge your input by running the timer on Clockify.me, then all your working time down to the second will automatically be reported to me for record keeping.

**Comment by gtainfotech** [comment]
@prolay7, @lFighterl is working on the UI. Can anyone else help out?

**Comment by JaspreetKaur27** [comment]
yeah, I can help with UI.

**Comment by gtainfotech** [comment]
OK. Thanks @JaspreetKaur27.

**Comment by JaspreetKaur27** [comment]
how can I get to know what are their plans or they haven't started yet?

**Comment by gtainfotech** [comment]
This is a great place to ask. lFighterl told me he's going to get started just three or four hours ago.

**Comment by gtainfotech** [comment]
I am sure he will get back to us tomorrow.

**Comment by lFighterl** [comment]
Hii, I just wanted to if anyone has already started if they can push their files so we can start building from them rather than having different base code each. Also, let's discuss what section is being developed by whom, so we don't end up having developed the same page by 2 or more.

**Comment by gtainfotech** [comment]
So far Miss Jaspreet, Mr. Fighter and Mr. Prolay have shown interest in starting things off. So please discuss among the three of you, till each one owns a part of the app. Looks like the rest are going to wait for now.

**Comment by JaspreetKaur27** [comment]
I haven't started anything as I was waiting for you responses. so which tech are we using or starting from html and css?

**Comment by gtainfotech** [comment]
I am still headhunting so keep in mind that you may have to say good bye to parts of the code that you are going to own now. So that's cool I hope.

**Comment by gtainfotech** [comment]
We are using HTML, CSS, ReactJS, ExpressJS, NodeJS, Cassandra DB, GraphQL ...

**Comment by JaspreetKaur27** [comment]
Do we have any base code to start or its from scratch?

**Comment by JaspreetKaur27** [comment]
So I created new branch in which I create go-for-it react app. Please let me know if that is okay?
so we can discuss further about designing plan and section division.

**Comment by gtainfotech** [comment]
It's going to be done all from scratch.

**Comment by gtainfotech** [comment]
Sure. The branch "initial base code" (if I remembered correctly) is great.

**Comment by JaspreetKaur27** [comment]
yeah. so I will wait for Mr. Prolay and Mr. Fighter for discussing section development division.

**Comment by gtainfotech** [comment]
Jaspreet, I have a question. Is it okay to have an Alpha, a Beta and a Release folder in your branch, and the pretest development should be done in Alpha directory. I decided on this myself. Later we can price their usage online by end users differently.

**Comment by gtainfotech** [comment]
@JaspreetKaur27, So we are going to have all our initial development done in the Alpha directory. I went ahead and fixed your branch structure. I hope that's alright with you.

**Comment by gtainfotech** [comment]
Miss Jaspreet, please, check the newly created branch by Mr. lFighterl for latest done. Still, you two need to divide up the workload.

**Comment by JaspreetKaur27** [comment]
yeah It's alright but I was thinking as we need to use express so I think we need to create express app like backend side and then inside it react app. Does that sound correct??

**Comment by JaspreetKaur27** [comment]
I created new branch and delete old ones. Edit app.js and app.css to test application. Display welcome message and did some css on it.

Now need to divide the work according to mockup design.

**Comment by gtainfotech** [comment]
@JaspreetKaur27 & @lFighterl, could you please rename the directories named go-for-it-pass in your Alpha directories to the version number: 0.0.1 instead of go-for-it-pass?
So this way we can keep track of the versions.

**Comment by gtainfotech** [comment]
@lFighterl, we need to change the master branch from /Alpha/go-for-it-pass to /Alpha/0.0.1 too. I hope this can be done.

**Comment by prolay7** [comment]
Hi, Jaspreet, have you started yet with the UI with express

**Comment by prolay7** [comment]
Let me know

**Comment by prolay7** [comment]
Jaspreet let me know

**Comment by gtainfotech** [comment]
@prolay7, JaspreetKaur27 is in Toronto so she is asleep right now. Please wait till tomorrow.

**Comment by JaspreetKaur27** [comment]
Hi , I just edit the message on home screen. we need to discuss how we will do and which section will do by whom. you can merge my branch and check what I did. It's just message on home screen.

**Comment by JaspreetKaur27** [comment]
Do we need to follow UI mockup details??

**Comment by lFighterl** [comment]
initial base code is your branch right ?

**Comment by lFighterl** [comment]
i only have node modules in that branch, is there any other files ?

**Comment by lFighterl** [comment]
i don't know express.js so if you could set up react in it and let me know what's the working directory

**Comment by gtainfotech** [comment]
Mockup details are important, but not must follow ones. If you have better ideas, I think we would all love to see them.
Miss @JaspreetKaur27, can you please set up express and let @lFighterl know what the working directory is?
Also, as mentioned before, please, change the names of your directory from go-for-it-pass to a version number (0.0.1) inside the Alpha directory.

**Comment by JaspreetKaur27** [comment]
okay, first let me set up the express and react app. I will change the directory name also,

**Comment by lFighterl** [comment]
thanks

**Comment by gtainfotech** [comment]
Miss @JaspreetKaur27, sorry about my ignorance of GitHub, but is there a way to test your change to the welcome message in your branch first on my local machine, before merging it with master branch?
Thanks for the help.

**Comment by gtainfotech** [comment]
I have GitHub Desktop, and am going to set up a local machine. Maybe I have to wait for Mr. @lFighterl to do all his research to see the details of the tech stack we need. Generally speaking the stack is going to look like this:
Cassandra DB, Express, React, Node, (later) React Native and Electron for mobile and desktop versions, and if there is a need we can use GraphQL.

**Comment by JaspreetKaur27** [comment]
yeah you can test it. Don't merge it in master yet but you can test at you local machine

**Comment by gtainfotech** [comment]
OK, Thanks.

**Comment by JaspreetKaur27** [comment]
I am creating a directory for version (0.0.1) in Alpha and in that I will set up react and express.

**Comment by gtainfotech** [comment]
OK.

**Comment by JaspreetKaur27** [comment]
Hey, I updated my branch (landing-page1). Working repo is inside Alpha -> 0.0.1 -> Server -> client.

the client folder is react app where we gonna work on UI.

**Comment by gtainfotech** [comment]
OK.

**Comment by lFighterl** [comment]
Thank you, i have a thing setup i'll transfer them

**Comment by JaspreetKaur27** [comment]
On master I can't see any working repo except node_modules in client folder.

**Comment by gtainfotech** [comment]
Miss @JaspreetKaur27, is everything OK now?

**Comment by ajayguru** [comment]
Hey, I guess someone has mistakenly committed "**node_modules**" folder!

**Comment by gtainfotech** [comment]
@ajayguru, you mean that folder shouldn't be committed? How could I exclude it from being committed when it's in the pull request?
Can you send me a pull request with the correct folders in it?

**Comment by gtainfotech** [comment]
Yeah, I am a newbie. Sorry.

**Comment by gtainfotech** [comment]
Is it enough to just put node_modules in .gitignore?

**Comment by JaspreetKaur27** [comment]
Did you add node_modules in .gitignore file or not??

**Comment by JaspreetKaur27** [comment]
If not I can do that and can test it

**Comment by gtainfotech** [comment]
I see that Ms. @JaspreetKaur27 has done it. It's now pending peer review.

**Comment by gtainfotech** [comment]
@ajayguru approved and I merged it. Please, guys, check the communication lines more often. GitHub, Skype both.

**Comment by ajayguru** [comment]
Hi Team,
I have a couple of following suggestions:-
1) We should go for "SCSS" instead of plain CSS.
2) We should choose Material-UI/Semantic-UI instead of Bootstrap.

Please feel free to provide your valuable feedback!

**Comment by gtainfotech** [comment]
How about SASS everyone? Thought I should throw that in the middle, too.

**Comment by lFighterl** [comment]
scss is saas, and we can use saas as long as everyone knows how to use it. and i dont know much about semantic ui or material ui, can you explain its uses a little ?

**Comment by JaspreetKaur27** [comment]
I think we can follow documentation to implement UI. I know about Material-UI but not semantic. I think, We can go with Material UI.

**Comment by gtainfotech** [comment]
So is the following okay with everyone for Tech stack: Cassandra DB, ExpressJS, ReactJS, ReduxJS, NodeJS, GraphQL, SCSS, Material UI ...

**Comment by gtainfotech** [comment]
I just am not sure about Material UI. I see a lot of advantages to using Semantic UI.

**Comment by gtainfotech** [comment]
If everyone is okay with Tech stack: Cassandra DB, ExpressJS, ReactJS, ReduxJS, NodeJS, GraphQL, SCSS, Semantic UI, it will be settled.

**Comment by ajayguru** [comment]
> I just am not sure about Material UI. I see a lot of advantages to using Semantic UI.
[Material UI](https://material-ui.com/) is also a popular framework which has most inbuilt ready to use UI component but a bit heavy in terms of performance as compare to Semantic-UI.

**Comment by kenzdozz** [comment]
The Tech stack is okay. But in the project "server" directory, I see "Jade", is there going to be any server side rendering?

**Comment by gtainfotech** [comment]
> But in the project "server" directory, I see "Jade", is there going to be any server side rendering?

If there is going to be any, it's best for it to be react server side rendering.

**Comment by JaspreetKaur27** [comment]
> The Tech stack is okay. But in the project "server" directory, I see "Jade", is there going to be any server side rendering?

No there is no server side rendering. that folder come when we setup express app for backend we can remove that folder.
